docs/changelog/changelog_en.md

```diff
@@ -98,6 +98,7 @@
 
 ### 👷 Changed
 
+- Change the mobile design of the cookie bar layout
```

Comment:
this is old version, please create a `### 👷 Changed` category in the newest version and put the changelog there.